lib/core/renderable/LocalPlayer.simba

```diff
@@ -1,13 +1,10 @@
 
 procedure TReflectLocalPlayer.Create;
 begin
-  if HookCache[TCacheKey.ckLocalPlayer][0] = 0 then
```

Comment:
The changed code here is essentially the same, I just cleaned up the code as 'Reflect.Mem.GetObject(ckNull, ckLocalPlayer, Client_LocalPlayer, 0, 0)' was being called in either if and else statements.